db/views/patient_summaries_v05.sql

```diff
@@ -0,0 +1,116 @@
+next_appointment_facility.name AS next_appointment_facility_name,
+next_appointment_facility.facility_type AS next_appointment_facility_type,
+next_appointment_facility.district AS next_appointment_district,
+next_appointment_facility.state AS next_appointment_state,
+
+(
+    CASE
+      WHEN next_appointment.scheduled_date IS NULL THEN 0
+      WHEN next_appointment.scheduled_date > date_trunc('day', NOW() - interval '30 days') THEN 0
+      WHEN (latest_blood_pressure.systolic >= 180 OR latest_blood_pressure.diastolic >= 110) THEN 1
+      WHEN (
+        (mh.prior_heart_attack = 'yes' OR mh.prior_stroke = 'yes')
+        AND (latest_blood_pressure.systolic >= 140 OR latest_blood_pressure.diastolic >= 90)
+      ) THEN 1
+      WHEN (
+        (latest_blood_sugar.blood_sugar_type = 'random' AND latest_blood_sugar.blood_sugar_value >= 300)
+        OR (latest_blood_sugar.blood_sugar_type = 'post_prandial' AND latest_blood_sugar.blood_sugar_value >= 300)
+        OR (latest_blood_sugar.blood_sugar_type = 'fasting' AND latest_blood_sugar.blood_sugar_value >= 200)
+        OR (latest_blood_sugar.blood_sugar_type = 'hba1c' AND latest_blood_sugar.blood_sugar_value >= 9.0)
+      ) THEN 1
+      ELSE 0
+    END
+) AS risk_level,
+
+latest_bp_passport.identifier AS latest_bp_passport,
+p.id
+
+FROM patients p
+
+LEFT OUTER JOIN addresses ON addresses.id = p.address_id
+LEFT OUTER JOIN facilities reg_facility ON reg_facility.id = p.registration_facility_id
+LEFT OUTER JOIN medical_histories mh ON mh.patient_id = p.id
+
+LEFT OUTER JOIN (
+    SELECT DISTINCT ON (patient_id) *
+    FROM patient_phone_numbers
+    ORDER BY patient_id, device_created_at DESC
+) AS latest_phone_number
+ON latest_phone_number.patient_id = p.id
+
+LEFT OUTER JOIN (
+    SELECT DISTINCT ON (patient_id) *
+    FROM blood_pressures
+    ORDER BY patient_id, recorded_at DESC
+) AS latest_blood_pressure
+ON latest_blood_pressure.patient_id = p.id
+LEFT OUTER JOIN facilities latest_blood_pressure_facility ON latest_blood_pressure_facility.id = latest_blood_pressure.facility_id
+
+LEFT OUTER JOIN (
+    SELECT DISTINCT ON (patient_id) *
+    FROM blood_sugars
+    ORDER BY patient_id, recorded_at DESC
+) AS latest_blood_sugar
+ON latest_blood_sugar.patient_id = p.id
+LEFT OUTER JOIN facilities latest_blood_sugar_facility ON latest_blood_sugar_facility.id = latest_blood_sugar.facility_id
+
+LEFT OUTER JOIN (
+    SELECT DISTINCT ON (patient_id) *
+    FROM patient_business_identifiers
+    WHERE identifier_type = 'simple_bp_passport'
+    ORDER BY patient_id, device_created_at DESC
+) AS latest_bp_passport
+ON latest_bp_passport.patient_id = p.id
+
+LEFT OUTER JOIN (
+    SELECT DISTINCT ON (patient_id) *
+    FROM appointments
+    ORDER BY patient_id, scheduled_date DESC
+) AS next_appointment
+ON next_appointment.patient_id = p.id
+LEFT OUTER JOIN facilities next_appointment_facility ON next_appointment_facility.id = next_appointment.facility_id
+WHERE p.deleted_at IS NULL;
```

Comment:
This is the only change in the definition?

Reply:
Correcto 👌